### ISSUE TITLE
Linked Jutsu Design

### ISSUE THREAD

**Issue by PrincessSuper** [levimeahan/shinobi-chronicles#95]
Idea I had for jutsu that could be cool for SC. Linked ones, wherein you must have jutsu 1 on cooldown to use jutsu 2.

For example:

**Gaia's Prison**
Description: Trap your opponent in an earthy prison.
Power: 3
Effects: Speed Nerf-2 turns, Linked [Atlas' Revenge]
Cooldown: 2 turns

**Atlas' Revenge**
Description: Crush the earthen prison around your opponent.
Power: 4.2
Effects: Dependant [Gaia's Prison]

Hopefully that makes enough sense you can use the idea.

**Comment by levimeahan** [comment]
We actually have this implemented for Genjutsu - I guess the idea would be to do this for all types of jutsu?

**Comment by PrincessSuper** [comment]
I would say yes, it sets up a lot of new design space for techniques, and allows for essentially inverse cooldowns to make sure the combat balance stays fresh

**Comment by ViscaBarre** [comment]
I second the appeal.  Linked jutsus could very well be a way to implement elemental combinations (eg. turn 1 use water, turn 2 use fire, turn 3 a special steam jutsu becomes available).

**Comment by levimeahan** [comment]
Yeah, I think it makes sense to make this an explicit connection. Though I think for elemental combos I want to make that its own thing, so less "you can't use jutsu B before jutsu A" but rather "you only get the special interaction if you use Jutsu B after Jutsu A" e.g. water wave => lightning wave = increased damage
